flight/UAVObjects/uavobjectmanager.c

```diff
@@ -377,7 +397,7 @@ UAVObjHandle UAVObjRegister(uint32_t id,
  */
 UAVObjHandle UAVObjGetByID(uint32_t id)
 {
-	UAVObjHandle * found_obj = (UAVObjHandle *) NULL;
+	UAVObjHandle found_obj = NULL;
```

Comment:
Also, if you've not encountered the command line tool 'cdecl' it's very helpful in explaining what declarations mean 
```cdecl> explain void *(*a)(void *)
declare a as pointer to function (pointer to void) returning pointer to void```

This is particularly helpful with things like array arguments.